app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  after_create :update_posts_counter
```

Comment:
- **[OPTIONAL]** We can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_post_counter` method every time a new post has been saved to database, then make the `update_post_counter` method private.

```
  after_save :update_post_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end
```